### ISSUE TITLE
Release type

### ISSUE THREAD

**Issue by Sildenafil99** [vitiko98/qobuz-dl#216]
Is it possible to split downloads by release type when downloading an artist's discography? (Album, EP & Singles, Other)

ex. Artist\ReleaseType\AlbumName\TrackTitle

thx

**Comment by cmd64** [comment]
Yeah, it is possible. You need to do some changes download_release and download_track methods at downloader.py

download_release
find this line
`dirn = os.path.join(self.path, sanitized_title)`

replace with this
```
release_type = meta.get("release_type").upper()
dirn = os.path.join(self.path, release_type, sanitized_title)
```

download_track
find this line
`dirn = os.path.join(self.path, sanitized_title)`

replace with this
```
release_type = meta["album"]["release_type"].upper()
dirn = os.path.join(self.path, release_type, sanitized_title)
```


You need to make build again. Download source code and move C:\qobuz-dl-master. Open terminal as administrator.
cd C:\qobuz-dl-master
py -m build
pip install C:\qobuz-dl-master\dist\qobuz_dl-0.9.9.10-py3-none-any.whl --force-reinstall --no-cache-dir

**Comment by Dreamkiller711** [comment]
Is there a way to add the release type to the folder_format line in config.ini ? I'd like the choice to add something like {release_type} anywhere in the folder structure.
